src/contractTest/java/uk/gov/hmcts/futurehearings/hmi/cdc/consumer/common/TestingUtils.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.futurehearings.hmi.cdc.consumer.common;
```

Comment:
My understanding was that this class was made common...why is this now sitting in consumer/common.
Wasn't your refactoring merged to master and then rebased to this branch?If so please do so....